azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/DeployMojo.java

```diff
@@ -56,7 +56,7 @@
      *
      * @since 0.1.0
      */
-    @Parameter(property = "functions.deploymentType", defaultValue = DeploymentType.ConstantValues.MS_DEPLOY_VALUE)
+    @Parameter(property = "functions.deploymentType", defaultValue = DeploymentType.ConstantValues.ZIP_VALUE)
```

Comment:
We talked about the v2 schema yesterday. Do we want to use ZIPDEPLOY by default for v1 schema users?

Reply:
I think the default value could align between schema v1 and v2. We will release the zip deploy with the documents telling users about this change, and it will know breaking the deployment experience for current users.